oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsUsbDevice.java

```diff
@@ -133,7 +133,7 @@ private static WindowsUsbDevice queryDeviceAndChildren(Integer device, Map<Integ
             }
         }
         // Iterate the parent map looking for children
-        Set<Integer> childDeviceSet = parentMap.entrySet().stream().filter(e -> e.getValue().equals(device))
+        Set<Integer> childDeviceSet = parentMap.entrySet().stream().parallel().filter(e -> e.getValue().equals(device))
```

Comment:
Filtering a map entryset in parallel could slightly improve performance but this would result in a random order of USB device listing each time someone ran it, so probably not worth it.